resources/js/alpine/easyMde.js

```diff
@@ -0,0 +1,49 @@
+/* EasyMDE */
+
+import EasyMDE from 'easymde'
+import DOMPurify from 'dompurify'
+
+export default () => ({
+  init() {
+    this.$nextTick(() => {
+      new EasyMDE({
+        element: this.$el,
```

Comment:
```suggestion
        element: this.$el,
        forceSync: true,
        spellChecker: false,
```
Думаю стоит отключить проверку орфографии, эта библиотека тянет с cdn файлы.

Reply:
> Думаю стоит отключить проверку орфографии, эта библиотека тянет с cdn файлы.

Согласен отключить проверку орфографии, но сделать доступной её через параметр в будущем.

Reply:
можно глянуть на реализацию https://github.com/moonshine-software/moonshine/blob/d124d170e45a9f5f99f6928053c7701a98ae2108/resources/js/alpine/tinymce.js#L37

если получится, вынести из tinymce и сделать более универсальным, в choicesjs это тоже можно будет использовать